resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/DriverInitContainerComponentsProvider.scala

```diff
@@ -105,7 +105,7 @@ private[spark] class DriverInitContainerComponentsProviderImpl(
   private val dockerImagePullPolicy = sparkConf.get(DOCKER_IMAGE_PULL_POLICY)
   private val downloadTimeoutMinutes = sparkConf.get(INIT_CONTAINER_MOUNT_TIMEOUT)
 
-  override def provideInitContainerConfigMapBuilder(
+  private def provideInitContainerConfigMapBuilder(
```

Comment:
This is causing Compile errors in Unit Test (as seen [here](http://spark-k8s-jenkins.pepperdata.org:8080/job/PR-spark-k8s-full-build/582/console) in ClientV2Suite) please fix